backend/src/validators/users.js

```diff
@@ -30,6 +30,17 @@ const addOne = {
   response: getAddResponse(),
 };
 
+const getByEmail = {
+  headers: S.object().prop("Authorization", S.string()),
```

Comment:
Lets make sure we include the schema information in here, this should be a quick fix.